layers/+misc/nlinum/funcs.el

```diff
@@ -0,0 +1,20 @@
+;;; packages.el --- nlinum Layer packages File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Thomas de Beauchêne <thomas.de.beauchene@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs/nlinum-on ()
+  (when (spacemacs/linum-enabled-p)
+    (nlinum-mode 1)))
+
+(defun spacemacs/nlinum-relative-toggle ()
+  (interactive)
+  (if (not (bound-and-true-p nlinum-relative-mode))
+      (nlinum-mode))
+  (nlinum-relative-toggle))
```

Comment:
Nope, it is supposed to be like that :smile_cat: 

You want to toggle the toggle every time it is called, but turn on line numbers only if they were not already on, right ?